cmd/root.go

```diff
@@ -396,7 +393,7 @@ func NewCommand(opts ...Option) *Command {
 		Use:     "cloud-sql-proxy INSTANCE_CONNECTION_NAME...",
 		Version: versionString,
 		Short:   "cloud-sql-proxy authorizes and encrypts connections to Cloud SQL.",
-		Long:    longHelp,
+		Long:    strings.ReplaceAll(longHelp, "{x-release-please-version}", ""), //remove the inline annotation required by release-please to update version.
```

Comment:
Could we move this comment above this line, just to help with line length?

Reply:
updated.